react-stopwatch-component/src/stopwatch.jsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+
+class Stopwatch extends React.Component{
+  constructor(props){
+    super(props);
+    this.state = {
+      count: 0,
+      pause: false
+    }
+
+    this.tick = this.tick.bind(this);
+    this.add = this.add.bind(this);
+    this.reset = this.reset.bind(this);
+    this.pause = this.pause.bind(this);
+    this.timeActions = this.timeActions.bind(this);
+    let intervalId = '';
```

Comment:
the only change I would do is change this to `this.intervalId` so you create the property now instead of in the tick method